app/assets/javascripts/accounts.coffee

```diff
@@ -1,3 +1,9 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+$(document).ready ->
+  $('.ui.checkbox').click ->
+    if($(@).hasClass('checked'))
```

Comment:
kkkkkkkk nem encontrei no google também... Não seria melhor utilizar logo o $(this) pra não ter dúvida?